fingerprintauthhelper/src/main/kotlin/com/pro100svitlo/fingerprintAuthHelper/FahManager.kt

```diff
@@ -280,16 +263,20 @@ internal class FahManager(@NonNull c: Context, l: FahListener?, keyName: String,
                 }
             }
 
+            val keyStore: KeyStore
```

Comment:
This was done to avoid unnecessary null checks. As the keyStor property is mutable (var instead of val) it could possibly change from being not-null to null. Agreed, this shouldn't really happen, but the compiler things it might, so this is just to avoid warnings, essentially.